spaceship/lib/spaceship/client.rb

```diff
@@ -447,11 +441,10 @@ def send_shared_login_request(user, password)
         end
 
         response = request(:post) do |req|
-          req.url "https://idmsa.apple.com/appleauth/auth/signin"
-          req.body = data.to_json
-          req.headers['Content-Type'] = 'application/json'
-          req.headers['X-Requested-With'] = 'XMLHttpRequest'
-          req.headers['X-Apple-Widget-Key'] = self.itc_service_key
+          req.url "https://idmsa.apple.com/IDMSWebAuth/authenticate"
```

Comment:
@taquitos WRT to the hardcoded appID, could we add this as documentation?

# GET https://developer.apple.com/account/ returns a 302 redirect to idmsa.apple.com...

Reply:
Also there was a question from @youtux WRT encoding passwords.

https://github.com/fastlane/fastlane/pull/10826/files/8bf8bb3d3c4232356a035d4e8209756ec6a70c1f#r149506222